.sass-lint.yml

```diff
@@ -29,7 +29,7 @@ rules:
   no-empty-rulesets: 2
   no-extends: 0
   no-ids: 2
-  no-important: 2
+  no-important: 0
```

Comment:
The no-important seems to be a point of disagreement.  I'm inclined to agree with @AllenBW on this.  Question for @jeff-phillips-18 why would we be overriding things, if your team is doing the UX (it would seem that in that use case, there would be no need to override)?  Maybe I'm missing something.